lib/driverProviders/attachSession.ts

```diff
@@ -25,10 +24,10 @@ export class AttachSession extends DriverProvider {
    * @return {q.promise} A promise which will resolve when the environment is
```

Comment:
You can remove all these type annotations from the JSDoc, since this is now in TypeScript.

Reply:
Updated.